README.md

```diff
@@ -1,2 +1,62 @@
 # install-scripts
 Installation scripts for OpenShift KNI clusters
+
+## CNV
+The CNV installation is managed by a 'meta operator' called the [HyperConverged
+Cluster Operator](https://github.com/kubevirt/hyperconverged-cluster-operator) (HCO).
+Deploying with the 'meta operator' will launch operators for KubeVirt,
+Containerized Data Imported (CDI), Cluster Network Addons (CNA),
+Common templates (SSP), KubeVirt Web UI, and Node Maintenance Operator (NMO).
+
+#### Deploy without OLM
+```bash
+$ curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/deploy/deploy.sh | bash
+```
+
+#### Deploy CNV through Marketplace
+```bash
+$ curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/tools/quay-registry.sh | CLUSTER=OPENSHIFT bash -s $QUAY_USERNAME $QUAY_PASSWORD
+```
+
+After pointing Marketplace at the new app registry, you can go the rest of the
+way using the UI by finding KubeVirt Hyperconverged Cluster Operator in the
+OperatorHub tab.
+
+If you want to use the CLI, run the following commands:
+
+```bash
```

Comment:
Let's move this out into a script, in this repo or in hco

Reply:
:+1: or maybe this can just be a manifest file.